### DISCUSSION TITLE
How to use the convert_camel_case functionality in version 5

### DISCUSSION THREAD

**Discussion by alvaro-garcia-carrasco** [bcb/jsonrpcserver#215] [Q&A]
Hi! I am very excited and happy about the new version 5 release. I rely on the convert_camel_case option in the dispatched function, and I wonder if there is some way to get around it in the new version 5. 

We don't use the camel_case convention directly because we define the specification with open-rpc and we have the client being built in other languages. Thank you for any tips!

**Comment by bcb** [comment]
Hi @alvaro-garcia-carrasco , that option was removed but you may get by renaming the method, like:
```python
@method(name=“MyMethod”)
def my_method():
    …
```
Does that work for you?

**Reply by alvaro-garcia-carrasco** [reply]
that works for the method name but not for the name of the parameters in the method, maybe the client will need to make sure to change the naming convention before sending the request

**Reply by bcb** [reply]
Ok, we can consider adding this feature back.

**Reply by bcb** [reply]
Created issue #216 for this.
